src/components/common/ActionModal.vue

```diff
@@ -146,6 +151,7 @@
                 v-else
                 color="primary"
                 value="Submit"
+                :disabled="browserWithLedgerSupport"
```

Comment:
why is this disabled if browser has ledger support?

Reply:
Oops. This is a typo. This button should be disabled if the browser does not have ledger support